src/applications/mhv-landing-page/mocks/api/feature-toggles/index.js

```diff
@@ -21,6 +22,10 @@ const generateFeatureToggles = (toggles = {}) => {
           name: 'mhv_landing_page_enable_va_gov_health_tools_links',
           value: mhvLandingPageEnableVaGovHealthToolsLinks,
         },
+        {
+          name: 'mhv_secondary_navigation_enabled',
+          value: mhvSecondaryNavEnabled,
+        },
```

Comment:
It irks me a little when the camelCase and snake_case feature toggle names aren't the same since it gives me yet another thing I have to remember. If they are the same, when it comes time to remove the feature toggle it will be an easy search.

Suggestion: use `mhv_secondary_navigation_enabled` and `mhvSecondaryNavigationEnabled` for feature toggle names exclusively when referencing this feature in code.

Reply:
Ok. Changed